phpBB/adm/style/acp_bbcodes.html

```diff
@@ -47,6 +47,14 @@ <h1>{L_ACP_BBCODES}</h1>
 	</dl>
 	</fieldset>
 
+	<fieldset>
+		<legend>{L_APPEARANCE}</legend>
+	<dl>
+		<dt><label for="bbcode_font_icon">{L_BBCODE_FONT_ICON}</label><br /><span>{L_BBCODE_FONT_ICON_EXPLAIN}</span></dt>
+		<dd><input type="text" name="bbcode_font_icon" id="bbcode_font_icon" value="{{ BBCODE_FONT_ICON }}" /></dd>
```

Comment:
If I take it correctly, `BBCODE_FONT_ICON` is a language key. Then it should be `{{ lang('BBCODE_FONT_ICON') }}`.
Other language entries would be replaced same way, like `{L_APPEARANCE}` would become `{{ lang('APPEARANCE') }}` and so on.